apps/datahub/src/app/search/record-preview-datahub/record-preview-datahub.component.html

```diff
@@ -1,42 +1,43 @@
-<div class="flex record-item">
-  <div class="publisher flex-shrink-0 mx-6 flex">
+<div class="flex cursor-pointer flex-wrap sm:flex-nowrap">
+  <div class="flex-shrink-0 w-full sm:w-52">
     <div
-      class="overflow-hidden bg-gray-100 rounded-lg w-full border border-gray-500"
+      class="overflow-hidden bg-gray-100 rounded-lg w-full border border-gray-300 h-36"
     >
       <gn-ui-record-thumbnail
         class="relative h-full w-full object-cover object-left-top"
         [thumbnailUrl]="record.thumbnailUrl"
       ></gn-ui-record-thumbnail>
     </div>
   </div>
-  <div class="content flex-grow mx-2 flex flex-col justify-between min-w-0">
+  <div
+    class="content flex-grow grid grid-cols-1 auto-rows-min justify-between min-w-0 pr-6 sm:pl-6 sm:pr-12 sm:grid-cols-2"
+  >
     <div
-      class="py-2 font-title text-21 truncate"
+      class="order-1 mb-3 mt-5 font-title text-21 line-clamp-2 sm:line-clamp-1 sm:col-span-2 sm:mt-2"
       style="font-size: 21px"
       [title]="record.title"
     >
       {{ record.title }}
     </div>
-    <div class="flex-grow abstract leading-relaxed mb-3 line-clamp-3">
+    <div
+      class="order-3 flex-grow abstract mt-4 mb-5 h-36 sm:mb-2 sm:h-[4.5rem] line-clamp-6 sm:line-clamp-3 sm:col-span-2 sm:order-2 sm:mt-0"
+    >
       {{ abstract }}
     </div>
-    <div class="links flex flex-row justify-between" style="height: 2rem">
-      <div class="text-gray-400">
-        {{ record.contact?.organisation }}
-      </div>
-      <div class="basis-1/2">
-        <mat-icon
-          *ngIf="isDownloadable"
-          class="material-icons-outlined text-gray-200 mr-4"
-          >cloud_download</mat-icon
-        >
-        <mat-icon
-          *ngIf="isViewable"
-          class="material-icons-outlined text-gray-200 mr-4"
-          >map</mat-icon
-        >
-      </div>
+    <div class="order-2 text-gray-400 sm:order-3 sm:row-end-4 uppercase">
```

Comment:
I think the organisation should be displayed on one line, at least in desktop mode, and could take more space in the row.
- before 
![Screenshot from 2022-04-13 18-33-14](https://user-images.githubusercontent.com/1491924/163252580-fc3dc592-869e-4ae1-a002-a9155147bc21.png)
- after
![Screenshot from 2022-04-13 18-33-09](https://user-images.githubusercontent.com/1491924/163252582-646e9300-c187-4b74-bb42-8d67bf980399.png)

Reply:
Ah, thanks for noticing, this. Should be fixed with the commit below.

Reply:
Ok, take care, if it's very long it should be an elipsis, confirmed by Lucile.
Thanks

Reply:
Yep, I added a `truncate` for the "desktop" layout and let "mobile" layout wrap on multiple lines, as one line is much shorter here and multiple lines don't "break" the design.